### ISSUE TITLE
Translation of labels and shared document icon

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#880]
**Problematic behavior**
- When the interface is in French, the button label must be in French.
If the interface is set to French, the text that appears in the `aria-label` attributes must be translated.
- If CSS is disabled, there is a loss of information regarding the icon.

**Possible Solution**
- If CSS is disabled, there is a loss of information regarding the icon.
Hide the icon with an `aria-hidden="true"` attribute so it is not announced by screen readers, nor visible if there is no CSS enabled on the page.
Add hidden text like `sr-only` or `visually-hidden` that provides information equivalent to what the icon conveys.

**Additional context/Screenshots**
https://www.ffoodd.fr/masquage-accessible-de-pointe/
